src/form.js

```diff
@@ -917,7 +917,7 @@ define([
                 this.mugMap[this.normalizeHashtag(mug.hashtagPath)] = mug;
             }
         },
-        fixBrokenReferences: function (mug) {
+        fixBrokenReferences: function () {
```

Comment:
Seems like it could make sense to re-introduce this as an parameter and make this function operate on expressions referencing this mug only if given. That seemed out of scope for these optimizations, so I didn't do it in this PR.